readme.md

```diff
@@ -21,39 +21,12 @@ It is designed for the programmers who arefamiliar with the conventional Web MVC
 ```lisp
 (defun wmvc:demo ()
   (interactive)
-  (let ((src `(
-               ,(propertize "Form Sample" 'face 'info-title-1) BR
-               "This is a sample template.\nA normal text is inserted as is." BR
```

Comment:
@kiwanami : I consider replacing `BR` by `-`
which is less *eye noisy*.

(we could have also `_`  for `" "`